sdk/schemaregistry/azure-data-schemaregistry-apacheavro/pom.xml

```diff
@@ -44,15 +44,33 @@
   </properties>
 
   <dependencies>
+    <dependency>
+      <groupId>com.azure</groupId>
+      <artifactId>azure-core-experimental</artifactId>
+      <version>1.0.0-beta.24</version> <!-- {x-version-update;com.azure:azure-core-experimental;dependency} -->
+    </dependency>
     <dependency>
       <groupId>com.azure</groupId>
       <artifactId>azure-data-schemaregistry</artifactId>
       <version>1.1.0-beta.1</version> <!-- {x-version-update;com.azure:azure-data-schemaregistry;current} -->
     </dependency>
     <dependency>
-      <groupId>com.azure</groupId>
-      <artifactId>azure-core-serializer-avro-apache</artifactId>
-      <version>1.0.0-beta.20</version> <!-- {x-version-update;com.azure:azure-core-serializer-avro-apache;dependency} -->
+      <groupId>org.apache.avro</groupId>
+      <artifactId>avro</artifactId>
+      <version>1.10.1</version> <!-- {x-version-update;org.apache.avro:avro;external_dependency} -->
+    </dependency>
```

Comment:
(not for this PR) We should check if we can use the core avro serializer package here. If we have to fix/change anything related to Avro, we have a centralized place to do it if we use the core package.

Reply:
Ah yeah. Originally it was referencing azure-core-avro-serializer but since we no longer extend from ObjectSerializer, I didn't know if that was moot. Maybe some logic can be aggregated there.  Created #27011